### PR TITLE
Ref to course catogary

### PR DESCRIPTION
**Pull request by b46xu**
Added a reference link under the course search bar on the Credit Tracker page.

### CODE REVIEW

File: src/views/CreditTracker.vue [hunk 1]
```diff
@@ -50,7 +56,10 @@
 			</div>
 			<PieChart :chartData="pieChartData" :chartTitle="pieChartTitle" style="width: 200px; display: inline-block" />
 		</div>
-	</div>
+		<div id="app">
+			<div v-html="raw" v-linkified />
+			</div>
+				</div>
```
Comment (andythsu) [inline src/views/CreditTracker.vue]: Why do we need these four lines?

File: src/views/CreditTracker.vue [hunk 2]
```diff
@@ -60,11 +69,15 @@ const { AcademicYears } = require('@/Backend/Database');
 const { CacheService } = require('@/Backend/Service');
 const Enums = require('@/Backend/Enums');
 
+
+
+
```
Comment (andythsu) [inline src/views/CreditTracker.vue]: delete these unnecessary blank lines

**Review by andythsu — CHANGES_REQUESTED** [review]

File: src/views/CreditTracker.vue [hunk 3]
```diff
@@ -23,6 +25,10 @@
 				</b-input-group>
 				<b-button variant="success" @click="addCourse">Add</b-button>
 			</div>
+			<div>
+			
+				Please check on  : <a href="https://ugradcalendar.uwaterloo.ca/" class="linkified" target="_blank">https://ugradcalendar.uwaterloo.ca/</a> for more course catogary infomation.
+				</div>
```
Comment (andythsu) [inline src/views/CreditTracker.vue:31]:
```
<div>
<b-form-select>...</b-form-select>
<label style="
    font-size: 12px;
    float: left;
">Please visit {{url}} for more info
</label>
</div>

```
You can change line [#24](https://github.com/andythsu/WaterlooStudies/pull/13/files#diff-b5be0abcfcab0ef1ff76b1776ce91a625d13552c3fff739a12cb136d3e86185fR24) to this to make it display better

**Review by andythsu — CHANGES_REQUESTED** [review]

**Comment by b46xu** [comment]
Problems resolved

File: package.json [hunk 4]
```diff
@@ -20,6 +20,7 @@
 		"treantjs": "^1.0.0",
 		"vue": "^2.6.12",
 		"vue-chartjs": "^3.5.1",
+		"vue-linkify": "^1.0.1",
```
Comment (andythsu) [inline package.json]: we don't need this

File: src/views/CreditTracker.vue [hunk 5]
```diff
@@ -21,7 +22,14 @@
 					<span style="margin: 0 1em">falls in</span> 
 					<b-form-select v-model="userCourseGroup" :options="courseGroups"></b-form-select>
 				</b-input-group>
-				<b-button variant="success" @click="addCourse">Add</b-button>
+				<div>
+					<label style="
+					font-size: 12px;
+					float: center;
+						">Please visit <a href="https://ugradcalendar.uwaterloo.ca/" class="linkified" target="_blank">https://ugradcalendar.uwaterloo.ca/</a> for more info
```
Comment (andythsu) [inline src/views/CreditTracker.vue]: we don't need `class=linkified`

File: src/views/CreditTracker.vue [hunk 6]
```diff
@@ -21,7 +22,14 @@
 					<span style="margin: 0 1em">falls in</span> 
 					<b-form-select v-model="userCourseGroup" :options="courseGroups"></b-form-select>
 				</b-input-group>
-				<b-button variant="success" @click="addCourse">Add</b-button>
+				<div>
+					<label style="
+					font-size: 12px;
+					float: center;
```
Comment (andythsu) [inline src/views/CreditTracker.vue:28]: should be `float:left`

**Review by andythsu — CHANGES_REQUESTED** [review]

**Review by andythsu — APPROVED** [review]

**Pull request merged**